<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QDialog" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RideShare - Sign In</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#SignIn {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                      stop:0 #0F3057, stop:0.33 #4D7EA8,
                      stop:0.66 #9DC5BB, stop:1 #E6F4F1);
}

QLabel {
    color: black;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
}

QLineEdit {
    border: 2px solid #4D7EA8;
    border-radius: 12px;
    padding: 5px 10px;
    background-color: rgba(255, 255, 255, 0.9);
    selection-background-color: #9DC5BB;
    font-size: 12px;
    color: black;
}

QPushButton#LoginButton {
    background-color: #0F3057;
    color: white;
    border-radius: 15px;
    font-weight: bold;
    padding: 8px 16px;
    border: none;
}

QPushButton#LoginButton:hover {
    background-color: #4D7EA8;
}

QPushButton#LoginButton:pressed {
    background-color: #9DC5BB;
}

QPushButton#ForgetPasswordButton {
    background-color: transparent;
    color: white;
    text-decoration: underline;
    border: none;
    text-align: left;
}

QPushButton#ForgetPasswordButton:hover {
    color: #9DC5BB;
}

QPushButton#CreateAccountButton {
    background-color: rgba(255, 255, 255, 0.2);
    color: white;
    border-radius: 15px;
    padding: 8px 16px;
    border: 1px solid white;
}

QPushButton#CreateAccountButton:hover {
    background-color: rgba(255, 255, 255, 0.3);
}
</string>
  </property>
  <widget class="QLabel" name="AppLogoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>311</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>SmartQuizGen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Welcome Back!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="UsernameContainer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="UsernameLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="UsernameLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UsernameEntry">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>36</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Enter your username</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="PasswordContainer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="PasswordLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="PasswordLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PasswordEntryLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="PasswordEntry">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Enter your password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SeeHidePassword">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/hide.png</normaloff>:/images/images/hide.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ForgetPasswordButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>420</y>
     <width>150</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QWidget" name="OrSeparatorContainer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>485</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="OrSeparatorLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="LeftLine">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="OrLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="RightLine">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="CreateAccountButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>520</y>
     <width>150</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackHomePage">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 0.2);
    border: none;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.3);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/back.png</normaloff>:/images/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
